reader.go

```diff
@@ -30,48 +30,58 @@ func (o *ReaderOptions) norm(name string) {
 
 // Reader reads data from a remote feed.
 type Reader struct {
-	remote *bfs.Object
-	opt    ReaderOptions
-	ctx    context.Context
-	num    int
+	io.Reader
 
-	br io.ReadCloser // bfs reader
-	cr io.ReadCloser // compression reader
-	fd FormatDecoder
+	remotes []*bfs.Object
+	closers []io.Closer
+	fd      FormatDecoder
```

Comment:
OK, I changed my mind, `fd` should be definitely in `streamReader` as we could "technically" have multiple remotes with different formats, e.g. `[file-a.pbz file-b.jsonz]`